specification/purview/data-plane/Azure.Analytics.Purview.Catalog/preview/2021-05-01-preview/purviewcatalog.json

```diff
@@ -2468,10 +2473,8 @@
             "$ref": "./examples/Discovery_Query_AssetType.json"
           },
           "Discovery_Query_Term": {
-            "$ref": "./examples/Discovery_Query_Term.json"
-          },
-          "Discovery_Query_TermOfGlossary": {
-            "$ref": "./examples/Discovery_Query_TermOfGlossary.json"
+            "$ref": "./examples/Discovery_Query_TermAssignment.json",
+            "description": "Find assets by the term assigned."
```

Comment:
This `description` element is invalid and I believe is why the breaking changes tool is failing.  Can you please remove it?

Reply:
Thanks. I have removed this line and will let arch aboard review and approve the breaking changes first.

Another question, do you know whether adding descriptions to each example is supported? If yes, how can I add this to the swagger file. If not, will we support this in the future? Providing example descriptions can make customer easier to understand which example can be referred to under different situations. Many thanks.